opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1569,10 +1569,10 @@ namespace detail {
             }
         }
 
-        const V sumSat = sw + so + sg;
-        sw = sw / sumSat;
-        so = so / sumSat;
-        sg = sg / sumSat;
+        //const V sumSat = sw + so + sg;
+        //sw = sw / sumSat;
+        //so = so / sumSat;
+        //sg = sg / sumSat;
```

Comment:
Have you verified that this rescaling is unnecessary for Norne and the SPE cases?